app/views/reports/index.html.erb

```diff
@@ -2,32 +2,41 @@
   <div class="container">
     <h1 class="title">Reports</h1>
 
+    <%= render_report 'total_tanks_by_facility',
+                      context_params: { organization_id: current_user.organization_id },
+                      actions: [] do |builder| %>
+      <div style="text-align: right">
+        <%= builder.export_csv_button 'Download CSV', class: 'button-primary' %>
+        <%= builder.export_xls_button 'Download Excel', class: 'button-primary' %>
+      </div>
```

Comment:
Mostly adding the buttons this way to show it can be done. We could easily just supply CSS for the button's generated by ReportsKit.